# QTL Analysis Results 

## Overview

This repository contains Quantitative Trait Loci (QTL) analysis results from multiple datasets, focusing on various contexts and biological systems. The data includes expression QTLs (eQTLs), protein QTLs (pQTLs), and splicing QTLs (sQTLs) across different brain regions and cell types.

## Dataset Breakdown
# xQTL Finemapping Results Summary

| Data Type | Cohort | Modality | Method | Path |
|-----------|--------|----------|--------|------|
| eQTL | KNIGHT | Brain | single_context_finemapping | analysis_result/single_context/KNIGHT_eQTL/export/summary/context_specific/Knight_eQTL_brain.exported.toploci.bed.gz |
| pQTL | KNIGHT | Brain | single_context_finemapping | analysis_result/single_context/KNIGHT_pQTL/export/summary/context_specific/Knight_pQTL_brain.exported.toploci.bed.gz |
| eQTL | MSBB | BM_10 | single_context_finemapping | analysis_result/single_context/MSBB_eQTL/export/summary/context_specific/BM_10_MSBB_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_22 | single_context_finemapping | analysis_result/single_context/MSBB_eQTL/export/summary/context_specific/BM_22_MSBB_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_36 | single_context_finemapping | analysis_result/single_context/MSBB_eQTL/export/summary/context_specific/BM_36_MSBB_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_44 | single_context_finemapping | analysis_result/single_context/MSBB_eQTL/export/summary/context_specific/BM_44_MSBB_eQTL.exported.toploci.bed.gz |
| pQTL | MSBB | BM_36 | single_context_finemapping | analysis_result/single_context/MSBB_pQTL/export/summary/context_specific/MSBB_BM36_pQTL.exported.toploci.bed.gz |
| eQTL | MetaBrain | Basalganglia | single_context_finemapping | analysis_result/single_context/MetaBrain/export/summary/context_specific/Metabrain_Basalganglia.exported.toploci.bed.gz |
| eQTL | MetaBrain | Cerebellum | single_context_finemapping | analysis_result/single_context/MetaBrain/export/summary/context_specific/Metabrain_Cerebellum.exported.toploci.bed.gz |
| eQTL | MetaBrain | Cortex | single_context_finemapping | analysis_result/single_context/MetaBrain/export/summary/context_specific/Metabrain_Cortex.exported.toploci.bed.gz |
| eQTL | MetaBrain | Hippocampus | single_context_finemapping | analysis_result/single_context/MetaBrain/export/summary/context_specific/Metabrain_Hippocampus.exported.toploci.bed.gz |
| eQTL | MetaBrain | Spinalcord | single_context_finemapping | analysis_result/single_context/MetaBrain/export/summary/context_specific/Metabrain_Spinalcord.exported.toploci.bed.gz |
| eQTL | MiGA | GFM | single_context_finemapping | analysis_result/single_context/MiGA_eQTL/export/summary/context_specific/MiGA_GFM_eQTL.exported.toploci.bed.gz |
| eQTL | MiGA | GTS | single_context_finemapping | analysis_result/single_context/MiGA_eQTL/export/summary/context_specific/MiGA_GTS_eQTL.exported.toploci.bed.gz |
| eQTL | MiGA | SVZ | single_context_finemapping | analysis_result/single_context/MiGA_eQTL/export/summary/context_specific/MiGA_SVZ_eQTL.exported.toploci.bed.gz |
| eQTL | MiGA | THA | single_context_finemapping | analysis_result/single_context/MiGA_eQTL/export/summary/context_specific/MiGA_THA_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | AC | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/AC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_10_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Ast_10_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Ast_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Ast_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Ast_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | DLPFC | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/DLPFC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Exc_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Exc_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Exc_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Exc_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Exc_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Exc_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Inh_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Inh_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Inh_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Inh_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Inh_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Inh_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_12_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Mic_12_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_13_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Mic_13_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Mic_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Mic_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Mic_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | OPC_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/OPC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | OPC_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/OPC_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | OPC_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/OPC_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Oli_DeJager | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Oli_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Oli_Kellis | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Oli_Kellis_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Oli_mega | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/Oli_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | PCC | single_context_finemapping | analysis_result/single_context/ROSMAP_eQTL/export/summary/context_specific/PCC_DeJager_eQTL.exported.toploci.bed.gz |
| pQTL | ROSMAP | DLPFC_Bennett | single_context_finemapping | analysis_result/single_context/ROSMAP_pQTL/export/summary/context_specific/DLPFC_Bennett_pQTL.exported.toploci.bed.gz |
| pQTL | ROSMAP | DLPFC_Klein_adjusted | single_context_finemapping | analysis_result/single_context/ROSMAP_pQTL/export/summary/context_specific/DLPFC_Klein_gpQTL_adjusted.exported.toploci.bed.gz |
| pQTL | ROSMAP | DLPFC_Klein_unadjusted | single_context_finemapping | analysis_result/single_context/ROSMAP_pQTL/export/summary/context_specific/DLPFC_Klein_gpQTL_unadjusted.exported.toploci.bed.gz |
| eQTL | ROSMAP | monocyte | single_context_finemapping | analysis_result/single_context/ROSMAP_pQTL/export/summary/context_specific/monocyte_ROSMAP_eQTL.exported.toploci.bed.gz |
| sQTL | ROSMAP | AC | single_context_finemapping | analysis_result/single_context/ROSMAP_sQTL/LeafCutter2/export/summary/context_specific/ROSMAP_AC_sQTL.exported.toploci.bed.gz |
| sQTL | ROSMAP | DLPFC | single_context_finemapping | analysis_result/single_context/ROSMAP_sQTL/LeafCutter2/export/summary/context_specific/ROSMAP_DLPFC_sQTL.exported.toploci.bed.gz |
| sQTL | ROSMAP | PCC | single_context_finemapping | analysis_result/single_context/ROSMAP_sQTL/LeafCutter2/export/summary/context_specific/ROSMAP_PCC_sQTL.exported.toploci.bed.gz |
| eQTL | STARNET | Mac | single_context_finemapping | analysis_result/single_context/STARNET_eQTL/export/summary/context_specific/STARNET_eQTL_Mac.exported.toploci.bed.gz |
| Multi | MSBB | All | multi_context_finemapping | analysis_result/multi_context/MSBB/export/summary/MSBB.exported.toploci.bed.gz |
| Multi | ROSMAP | All | multi_context_finemapping | analysis_result/multi_context/ROSMAP/export/summary/ROSMAP.exported.toploci.bed.gz |
| eQTL | MSBB | BM_10 | multi_gene_finemapping | analysis_result/multi_gene/MSBB/export/summary/context_specific/MSBB_BM_10_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_22 | multi_gene_finemapping | analysis_result/multi_gene/MSBB/export/summary/context_specific/MSBB_BM_22_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_36 | multi_gene_finemapping | analysis_result/multi_gene/MSBB/export/summary/context_specific/MSBB_BM_36_eQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM_44 | multi_gene_finemapping | analysis_result/multi_gene/MSBB/export/summary/context_specific/MSBB_BM_44_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | AC | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_AC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Ast_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Ast_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Ast_mega_eQTL.exported.toploci.bed.gz |
| pQTL | ROSMAP | DLPFC_Bennett | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_DLPFC_Bennett_pQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | DLPFC | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_DLPFC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Exc_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Exc_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Exc_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Exc_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Inh_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Inh_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Inh_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Inh_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Mic_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Mic_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Mic_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | OPC_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_OPC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | OPC_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_OPC_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Oli_DeJager | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Oli_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Oli_mega | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_Oli_mega_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | PCC | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_PCC_DeJager_eQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | monocyte | multi_gene_finemapping | analysis_result/multi_gene/ROSMAP/export/summary/context_specific/ROSMAP_monocyte_ROSMAP_eQTL.exported.toploci.bed.gz |
| eQTL | KNIGHT | Brain | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/Knight_eQTL.exported.toploci.bed.gz |
| pQTL | KNIGHT | Brain | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/Knight_pQTL.exported.toploci.bed.gz |
| eQTL | MSBB | BM10 | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MSBB_eQTL_BM10.exported.toploci.bed.gz |
| eQTL | MSBB | BM22 | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MSBB_eQTL_BM22.exported.toploci.bed.gz |
| eQTL | MSBB | BM36 | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MSBB_eQTL_BM36.exported.toploci.bed.gz |
| eQTL | MSBB | BM44 | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MSBB_eQTL_BM44.exported.toploci.bed.gz |
| pQTL | MSBB | BM36 | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MSBB_pQTL.exported.toploci.bed.gz |
| eQTL | MiGA | GFM | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MiGA_eQTL_GFM.exported.toploci.bed.gz |
| eQTL | MiGA | GTS | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MiGA_eQTL_GTS.exported.toploci.bed.gz |
| eQTL | MiGA | SVZ | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MiGA_eQTL_SVZ.exported.toploci.bed.gz |
| eQTL | MiGA | THA | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/MiGA_eQTL_THA.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_AC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_AC.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_Ast | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_Ast.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_DLPFC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_DLPFC.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_Exc | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_Exc.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_Inh | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_Inh.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_Mic | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_Mic.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_OPC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_OPC.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_Oli | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_Oli.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_PCC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_PCC.exported.toploci.bed.gz |
| eQTL | ROSMAP | DeJager_monocyte | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_monocyte.exported.toploci.bed.gz |
| pQTL | ROSMAP | DeJager | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_DeJager_pQTL.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_Ast | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_Ast.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_Exc | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_Exc.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_Inh | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_Inh.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_Mic | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_Mic.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_OPC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_OPC.exported.toploci.bed.gz |
| eQTL | ROSMAP | Kelli_Oli | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_Kelli_Oli.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_Ast | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_Ast.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_Exc | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_Exc.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_Inh | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_Inh.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_Mic | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_Mic.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_OPC | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_OPC.exported.toploci.bed.gz |
| eQTL | ROSMAP | mega_Oli | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_mega_Oli.exported.toploci.bed.gz |
| metabolome | ROSMAP | - | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/ROSMAP_metabolome.exported.toploci.bed.gz |
| eQTL | STARNET | Mac | trans_finemapping | analysis_result/finemapping_twas_trans/export/summary/context_specific/STARNET_eQTL.exported.toploci.bed.gz |
| GWAS | AD_Bellenguez_2022 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Bellenguez_2022.exported.toploci.bed.gz |
| GWAS | AD_Jansen_2021 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Jansen_2021.exported.toploci.bed.gz |
| GWAS | AD_Wightman_Full_2021 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Wightman_Full_2021.exported.toploci.bed.gz |
| GWAS | AD_Bellenguez_EADB_2022 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Bellenguez_EADB_2022.exported.toploci.bed.gz |
| GWAS | AD_Kunkle_Stage1_2019 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Kunkle_Stage1_2019.exported.toploci.bed.gz |
| GWAS | AD_Wightman_Excluding23andMe_2021 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Wightman_Excluding23andMe_2021.exported.toploci.bed.gz |
| GWAS | AD_Wightman_ExcludingUKBand23andME_2021 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Wightman_ExcludingUKBand23andME_2021.exported.toploci.bed.gz |
| GWAS | AD_Bellenguez_EADI_2022 | - | AD_GWAS_finemapping | analysis_result/AD_GWAS_finemapping/export/context_specific/AD_Bellenguez_EADI_2022.exported.toploci.bed.gz |
| QTL | ROSMAP | xQTL_only_colocalization | ColocBoost | analysis_result/ColocBoost/export/summary/xqtl_only_colocboost_export_filtered.bed.gz |
| QTL & GWAS | ROSMAP & AD | AD_xQTL_colocalization | ColocBoost | analysis_result/ColocBoost/export/summary/ad_xqtl_colocboost_export_filtered.bed.gz |
| GWAS | AD | AD_meta_colocalization | ColocBoost | analysis_result/ColocBoost/export/summary/ad_meta_colocboost_export_filtered.bed.gz |
| Gene & GWAS | ROSMAP & MSBB & AD | - | twas | analysis_result/twas/export/summary/FunGen_twas.exported.bed.gz |
| Gene & GWAS | ROSMAP & AD | - | ctwas | analysis_result/ctwas/export/summary/FunGen_ctwas.exported.bed.gz |
| mQTL & haQTL   | ROSMAP & MSSB & Knight     | Brain       | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/FunGen_xQTL_epi.bulk.exported.compact.bed.gz                  |
| haQTL    | ROSMAP      | DLPFC          | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/ROSMAP_DLPFC_haQTL.exported.toploci.compact.bed.gz |
| mQTL     | ROSMAP      | DLPFC          | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/ROSMAP_DLPFC_mQTL.exported.toploci.compact.bed.gz  |
| mQTL     | KNIGHT  | Brain             | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/KnightADRC_mQTL.exported.toploci.compact.bed.gz    |
| mQTL     | MSBB        | -              | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/MSBB_mQTL.exported.toploci.compact.bed.gz         |
| mQTL & haQTL   | ROSMAP & MSSB & Knight     | Brain       | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/FunGen_xQTL_epi.bulk.exported.bed.gz                  |
| haQTL    | ROSMAP      | DLPFC          | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/ROSMAP_DLPFC_haQTL.exported.toploci.bed.gz |
| mQTL     | ROSMAP      | DLPFC          | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/ROSMAP_DLPFC_mQTL.exported.toploci.bed.gz  |
| mQTL     | KNIGHT  | Brain             | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/KnightADRC_mQTL.exported.toploci.bed.gz    |
| mQTL     | MSBB        | -              | fSuSiE_finemapping | analysis_result/finemapping_twas/fsusie/export/summary/context_specific/MSBB_mQTL.exported.toploci.bed.gz         |
| snATAC   | ROSMAP      | Kelis_OPC       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_OPC.exported.toploci.bed.gz                  |
| snATAC   | ROSMAP      | Kelis_Exc       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_Exc.exported.toploci.bed.gz                  |
| snATAC   | ROSMAP      | Kelis_Ast       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_Ast.exported.toploci.bed.gz                  |
| snATAC   | ROSMAP      | Kelis_Oli       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_Oli.exported.toploci.bed.gz                  |
| snATAC   | ROSMAP      | Kelis_Mic       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_Mic.exported.toploci.bed.gz                  |
| snATAC   | ROSMAP      | Kelis_Inh       | fSuSiE_finemapping | analysis_result/snatac_fsusie/top_loci/context_specific/ROSMAP_snATAC_Kelis_Inh.exported.toploci.bed.gz                  |
| snATAC  & AD   | ROSMAP & AD      | Kelis Celltypes      | Coloc | analysis_result/coloc/fsusie_AD_coloc/snATAC_Kelis_AD_coloc_export_filtered.bed.gz                  |
| Bulk mQTL and haQTL  & AD   | ROSMAP & AD      | Brain      | Coloc | analysis_result/coloc/fsusie_AD_coloc/bulk_fsusie_AD_coloc_export_filtered.bed.gz                  |
| snATAC  & Gene   | ROSMAP      | Kelis Celltypes      | Coloc | analysis_result/coloc/fsusie_AD_coloc/snatac_ROSMAP.coloc.tsv.gz                  |





* AD_xQTL_colocalization results include "AD_Bellenguez_2022", "AD_Wightman_Excluding23andMe_2021", "AD_Wightman_ExcludingUKBand23andME_2021", "AD_Kunkle_Stage1_2019", "AD_Wightman_Full_2021"
## Usage Notes

1. All files are compressed (`.gz`) BED format  
2. Represents top loci from respective QTL analyses  
3. Suitable for downstream genomic and transcriptomic research  
4. For context definitions in each top loci table, see below: 

### Table Summary
- Each row represents a single variant associated with a gene in a specific context (e.g., tissue or study).

### Column Descriptions

| Column Name           | Type      | Description                                                                 |
|-----------------------|-----------|-----------------------------------------------------------------------------|
| `chr`                 | integer   | Chromosome number                                                           |
| `start`               | integer    | Genomic start coordinate (0-based)                                          |
| `end`                 | integer   | Genomic end coordinate (1-based)                                            |
| `a1`                  | character | Effect allele                                                               |
| `a2`                  | character | Reference allele                                                            |
| `variant_ID`          | character | Unique variant ID in format `chr:pos:ref:alt`                               |
| `gene_ID`             | character | Ensembl gene ID                                                             |
| `event_ID`            | character | Unique event ID for QTL signal (context + gene). Only present if lfsr passed |
| `cs_coverage_0.95`    | integer   | cs index if variant is in 95% credible set with minimum correlation purity > 0.8, else 0                          |
| `cs_coverage_0.7`     | integer   | cs index if variant is in 70% credible set with minimum correlation purity > 0.8, else 0                          |
| `cs_coverage_0.5`     | integer   | cs index if variant is in 50% credible set with minimum correlation purity > 0.8, else 0                          |
| `cs_coverage_0.95_purity0.5`    | integer   | When available, the CS index if the variant is in the 95% credible set with minimum correlation purity > 0.5; else 0 |
| `cs_coverage_0.7_purity0.5`    | integer   | When available, the CS index if the variant is in the 70% credible set with minimum correlation purity > 0.5; else 0 |
| `cs_coverage_0.5_purity0.5`    | integer   | When available, the CS index if the variant is in the 50% credible set with minimum correlation purity > 0.5; else 0 |
| `PIP`                 | double    | Posterior inclusion probability for causality                               |
| `conditional_effect`  | double    | Effect size from conditional model                                          |
| `lfsr`                | double    | Only in multi conetxt or multi gene results. Local False Sign Rate: the probability that the estimated effect has the wrong sign. Lower values indicate higher confidence in the direction of the effect. Variants without an lfsr value (i.e., NA) did not pass the filtering threshold in multi-context fine-mapping and are retained for reference only |                            |
 
Unique in ColocBoost Results

| Column Name       | Type      | Description                                                                                                             |
|-------------------|-----------|-------------------------------------------------------------------------------------------------------------------------|
| `event_ID`        | character | Trait combination that colocalizes within the same 95% colocalization confidence set (CoS).                             |
| `cos_ID`          | character | Unique identifier for each 95% colocalization confidence set (CoS).                                                    |
| `vcp`             | double    | Variant Colocalization Probability—estimated probability that a variant is shared among colocalized traits.            |
| `cos_npc`         | double    | Normalized probability of colocalization within a CoS, supporting shared vs. trait-specific signals.                   |
| `min_npc_outcome` | double    | Minimum normalized probability across traits in the CoS.                                                               |
| `coef`            | double    | Regression coefficient from the ColocBoost model for each trait.                                                       |

Unique in TWAS

| Column Name           | Type      | Description                                                                                                                                          |
|-----------------------|-----------|------------------------------------------------------------------------------------------------------------------------------------------------------|
| `start`               | integer   | Transcription start site - 1.                                                                                                                    |
| `end`                 | integer   | Transcription start site.                                                                                                                            |
| `TADB_start`          | integer   | Start coordinate of the extended TADB window used for cis-fine mapping ([TADB BED](https://github.com/cumc/xqtl-analysis/blob/main/resource/TADB_enhanced_cis.coding.bed)). |
| `TADB_end`            | integer   | End coordinate of the extended TADB window.                                                                                                          |
| `is_imputable`        | logical   | Whether the gene-context pair has predictive models with CV \( R^2 > 0.01 \) and \( p < 0.05 \) in at least one method.                              |
| `rsq_cv`              | double    | Cross-validated \( R^2 \) for expression prediction performance.                                                                                     |
| `pval_cv`             | double    | Cross-validation p-value for expression prediction performance.                                                                                      |
| `is_selected_method`  | logical   | Indicates if this method is the best (highest CV \( R^2 \) and \( p < 0.05 \)) for the gene–context pair.                                            |
| `block`               | character | LD block containing the gene’s TSS, based on predefined LD partitions.                                                                                |

Unique in cTWAS

| Column Name | Type    | Description                      |
|-------------|---------|----------------------------------|
| `start`     | integer | Transcription start site - 1 |
| `end`       | integer | Transcription start site         |

Unique in fSuSiE compact table
| Column Name          | Type     | Description                                                                 |
|----------------------|----------|-----------------------------------------------------------------------------|
| `maf`                | numeric  | Minor allele frequency                                                      |
| `cs_coverage_0.95`   | numeric  | Credible set coverage at 95% threshold                                     |
| `TADB_start`   | integer  | Cis-window start position, typically defined by TAD boundary               |
| `TADB_end`     | integer  | Cis-window end position, typically defined by TAD boundary                 |
| `effect_direction`   | integer  | Overall sign of the effect peak, based on `z` values                       |
| `effect_peak_start`  | integer  | Genomic start position of the effect peak                                  |
| `effect_peak_end`    | integer  | Genomic end position of the effect peak                                    |
| `effect_peak_index`  | integer  | Index of the effect peak in 1024-grid representation of the TAD            |
| `grid_position`      | integer  | Grid-based positions (x-axis) covered by the effect peak                   |
| `grid_resolution`    | integer  | Number of grid rows used (typically 512 or 1024)                            |
| `effect_size_z`      | numeric  | Unadjusted effect size (`z`) at each grid position                         |
| `cs_id`              | string   | ID of the 95% credible set within this context and TAD                     |
| `cs_root`            | string   | Unified root name for overlapping CS in the same context                   |

Unique in fSuSiE compact table 
| Column Name          | Type     | Description                                                                 |
|----------------------|----------|-----------------------------------------------------------------------------|
| `effect_peak_start`  | integer  | Genomic start position of the effect peak                                  |
| `effect_peak_end`    | integer  | Genomic end position of the effect peak                                    |
| `effect_peak_index`  | integer  | Index of the effect peak in 1024-grid representation of the TAD            |
| `grid_positions`      | numeric  | Comma‐separated list of numeric grid_position (Grid-based positions (x-axis) covered by the effect peak)                  |
| `grid_effects`      | string  | Comma‐separated list of numeric effect size (`beta`) at each grid position |
| `epi_mark_positions`   | string   | Comma‐separated list of genomics positions for the epigenetics marks that are covered by the estimated effect peak |
| `epi_mark_names`   | string   | Comma‐separated list of name for the epigenetics marks that are covered by the estimated effect peak                     |
| `epi_mark_effects`   | string   | Comma‐separated list of **interpolated** effect for the epigenetics marks that are covered by the estimated effect peak based on neighboring grid_effects. It should be aware that there is no 1 to 1 relationship between the grid effects and the actual epi mark effects|



### Notes

- This table may include results from multi-context or multi-gene analyses.
- Rows with missing `event_ID` indicate variants from multi-context analyses that did not pass the LFSR filtering threshold.
- The `cs_coverage_*` columns indicate membership in posterior credible sets of varying coverage (95%, 70%, 50%).


### Usage Tips

- Filter rows by `cs_coverage_0.95` or `PIP` to prioritize candidate causal variants.
- Restrict to rows with non-missing `event_ID` for significance-filtered analysis.
- For visualization, group by `context` and `gene_ID` to examine gene-level credible sets across studies.


# xQTL Association Results Summary
| Data Type | Cohort | Modality | Method | Path |
|-----------|--------|----------|--------|------|
| eQTL | KNIGHT | Brain | LR | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/eQTL/Brain/LR/ |
| eQTL | MiGA | GFM | LR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GFM/LR/ |
| eQTL | MiGA | GTS | LR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GTS/LR/ |
| eQTL | MiGA | SVZ | LR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/SVZ/LR/ |
| eQTL | MiGA | THA | LR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/THA/LR/ |
| eQTL | MSBB | BM_10 | LR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_10/LR/ |
| eQTL | MSBB | BM_22 | LR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_22/LR/ |
| eQTL | MSBB | BM_36 | LR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_36/LR/ |
| eQTL | MSBB | BM_44 | LR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_44/LR/ |
| eQTL | ROSMAP | AC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/AC/LR/ |
| eQTL | ROSMAP | DLPFC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/DLPFC/LR/ |
| eQTL | ROSMAP | monocyte | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/monocyte/LR/ |
| eQTL | ROSMAP | PCC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/PCC/LR/ |
| eQTL | ROSMAP | snuc_DeJager/Ast | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Ast/LR/ |
| eQTL | ROSMAP | snuc_DeJager/Exc | LR | analysis_result/marginal_significant_qtl/cisAssociational/ROSMAP/eQTL/snuc_DeJager/Exc/LR/ |
| eQTL | ROSMAP | snuc_DeJager/Inh | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Inh/LR/ |
| eQTL | ROSMAP | snuc_DeJager/Mic | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Mic/LR/ |
| eQTL | ROSMAP | snuc_DeJager/OPC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/OPC/LR/ |
| eQTL | ROSMAP | snuc_DeJager/Oli | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Oli/LR/ |
| eQTL | ROSMAP | snuc_Kellis/Ast | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Ast/LR/ |
| eQTL | ROSMAP | snuc_Kellis/Exc | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Exc/LR/ |
| eQTL | ROSMAP | snuc_Kellis/Inh | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Inh/LR/ |
| eQTL | ROSMAP | snuc_Kellis/Mic | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Mic/LR/ |
| eQTL | ROSMAP | snuc_Kellis/OPC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/OPC/LR/ |
| eQTL | ROSMAP | snuc_Kellis/Oli | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Oli/LR/ |
| eQTL | ROSMAP | snuc_mega/Ast | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Ast/LR/ |
| eQTL | ROSMAP | snuc_mega/Exc | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Exc/LR/ |
| eQTL | ROSMAP | snuc_mega/Inh | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Inh/LR/ |
| eQTL | ROSMAP | snuc_mega/Mic | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Mic/LR/ |
| eQTL | ROSMAP | snuc_mega/OPC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/OPC/LR/ |
| eQTL | ROSMAP | snuc_mega/Oli | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Oli/LR/ |
| eQTL | STARNET | - | LR | analysis_result/marginal_significant_qtl/cis_association/STARNET/eQTL/LR/ |
| eQTL | KNIGHT | Brain | msex interaction | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/eQTL/Brain/interaction/msex/ |
| eQTL | MiGA | GFM | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GFM/interaction/msex/ |
| eQTL | MiGA | GTS | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GTS/interaction/msex/ |
| eQTL | MiGA | SVZ | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/SVZ/interaction/msex/ |
| eQTL | MiGA | THA | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/THA/interaction/msex/ |
| eQTL | MSBB | BM_10 | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_10/interaction/msex/ |
| eQTL | MSBB | BM_22 | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_22/interaction/msex/ |
| eQTL | MSBB | BM_36 | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_36/interaction/msex/ |
| eQTL | MSBB | BM_44 | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_44/interaction/msex/ |
| eQTL | ROSMAP | AC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/AC/interaction/msex/ |
| eQTL | ROSMAP | DLPFC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/DLPFC/interaction/msex/ |
| eQTL | ROSMAP | monocyte | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/monocyte/interaction/msex/ |
| eQTL | ROSMAP | PCC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/PCC/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/Ast | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Ast/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/Exc | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Exc/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/Inh | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Inh/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/Mic | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Mic/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/OPC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/OPC/interaction/msex/ |
| eQTL | ROSMAP | snuc_DeJager/Oli | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Oli/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/Ast | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Ast/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/Exc | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Exc/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/Inh | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Inh/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/Mic | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Mic/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/OPC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/OPC/interaction/msex/ |
| eQTL | ROSMAP | snuc_Kellis/Oli | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Oli/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/Ast | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Ast/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/Exc | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Exc/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/Inh | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Inh/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/Mic | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Mic/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/OPC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/OPC/interaction/msex/ |
| eQTL | ROSMAP | snuc_mega/Oli | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Oli/interaction/msex/ |
| eQTL | STARNET | - | msex interaction | analysis_result/marginal_significant_qtl/cis_association/STARNET/eQTL/interaction/msex/ |
| eQTL | ROSMAP | AC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/AC/interaction/APOE/ |
| eQTL | ROSMAP | DLPFC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/DLPFC/interaction/APOE/ |
| eQTL | ROSMAP | monocyte | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/monocyte/interaction/APOE/ |
| eQTL | ROSMAP | PCC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/PCC/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/Ast | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Ast/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/Exc | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Exc/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/Inh | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Inh/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/Mic | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Mic/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/OPC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/OPC/interaction/APOE/ |
| eQTL | ROSMAP | snuc_DeJager/Oli | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_DeJager/Oli/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/Ast | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Ast/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/Exc | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Exc/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/Inh | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Inh/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/Mic | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Mic/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/OPC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/OPC/interaction/APOE/ |
| eQTL | ROSMAP | snuc_Kellis/Oli | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_Kellis/Oli/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/Ast | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Ast/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/Exc | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Exc/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/Inh | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Inh/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/Mic | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Mic/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/OPC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/OPC/interaction/APOE/ |
| eQTL | ROSMAP | snuc_mega/Oli | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Oli/interaction/APOE/ |
| pQTL | KNIGHT | Brain | LR | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/pQTL/Brain/LR/ |
| pQTL | MSBB | - | LR | analysis_result/marginal_significant_qtl/cis_association/MSBB/pQTL/LR/ |
| pQTL | ROSMAP | DLPFC | LR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/pQTL/DLPFC/LR/ |
| pQTL | KNIGHT | Brain | msex interaction | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/pQTL/Brain/interaction/msex/ |
| pQTL | MSBB | - | msex interaction | analysis_result/marginal_significant_qtl/cis_association/MSBB/pQTL/interaction/msex/ |
| pQTL | ROSMAP | DLPFC | msex interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/pQTL/DLPFC/interaction/msex/ |
| pQTL | ROSMAP | DLPFC | APOE interaction | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/pQTL/DLPFC/interaction/APOE/ |
| eQTL | KNIGHT | Brain | QR | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/eQTL/Brain/QR/ |
| eQTL | MiGA | GFM | QR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GFM/QR/ |
| eQTL | MiGA | GTS | QR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/GTS/QR/ |
| eQTL | MiGA | SVZ | QR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/SVZ/QR/ |
| eQTL | MiGA | THA | QR | analysis_result/marginal_significant_qtl/cis_association/MiGA/eQTL/THA/QR/ |
| eQTL | MSBB | BM_10 | QR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_10/QR/ |
| eQTL | MSBB | BM_22 | QR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_22/QR/ |
| eQTL | MSBB | BM_36 | QR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_36/QR/ |
| eQTL | MSBB | BM_44 | QR | analysis_result/marginal_significant_qtl/cis_association/MSBB/eQTL/BM_44/QR/ |
| eQTL | ROSMAP | AC | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/AC/QR/ |
| eQTL | ROSMAP | DLPFC | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/DLPFC/QR/ |
| eQTL | ROSMAP | monocyte | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/monocyte/QR/ |
| eQTL | ROSMAP | PCC | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/PCC/QR/ |
| eQTL | ROSMAP | snuc_mega/Ast | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Ast/QR/ |
| eQTL | ROSMAP | snuc_mega/Exc | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Exc/QR/ |
| eQTL | ROSMAP | snuc_mega/Inh | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Inh/QR/ |
| eQTL | ROSMAP | snuc_mega/Mic | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Mic/QR/ |
| eQTL | ROSMAP | snuc_mega/OPC | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/OPC/QR/ |
| eQTL | ROSMAP | snuc_mega/Oli | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/eQTL/snuc_mega/Oli/QR/ |
| pQTL | KNIGHT | Brain | QR | analysis_result/marginal_significant_qtl/cis_association/KNIGHT/pQTL/Brain/QR/ |
| pQTL | MSBB | - | QR | analysis_result/marginal_significant_qtl/cis_association/MSBB/pQTL/QR/ |
| pQTL | ROSMAP | DLPFC | QR | analysis_result/marginal_significant_qtl/cis_association/ROSMAP/pQTL/DLPFC/QR/ |

For details on the results, please refer to our [protocol](https://github.com/StatFunGen/xqtl-protocol/blob/main/code/association_scan/TensorQTL/TensorQTL.ipynb)
